ts/screens/modal/IdentificationModal.tsx

```diff
@@ -379,8 +359,40 @@ class IdentificationModal extends React.PureComponent<Props, State> {
     );
   };
 
+  private getErrorText = () => {
```

Comment:
Could you please write a comment by making explicit when we can see the retry message or not?